### ISSUE TITLE
The _imagingft C module is not installed

### ISSUE THREAD

**Issue by NicoleCollard** [timothycrosley/streamdeck-ui#79]
hi friends, i've a problem on my workstation with the streamdeck, would i add 'Text' insted of a picture the terminal will shown me:

```
Traceback (most recent call last):
  File "/home/nicole/.local/lib/python3.9/site-packages/streamdeck_ui/gui.py", line 44, in update_button_text
    api.set_button_text(deck_id, _page(ui), selected_button.index, text)
  File "/home/nicole/.local/lib/python3.9/site-packages/streamdeck_ui/api.py", line 146, in set_button_text
    render()
  File "/home/nicole/.local/lib/python3.9/site-packages/streamdeck_ui/api.py", line 269, in render
    image = _render_key_image(deck, **button_settings)
  File "/home/nicole/.local/lib/python3.9/site-packages/streamdeck_ui/api.py", line 293, in _render_key_image
    true_font = ImageFont.truetype(os.path.join(FONTS_PATH, font), 14)
  File "/home/nicole/.local/lib/python3.9/site-packages/PIL/ImageFont.py", line 640, in truetype
    return freetype(font)
  File "/home/nicole/.local/lib/python3.9/site-packages/PIL/ImageFont.py", line 637, in freetype
    return FreeTypeFont(font, size, index, encoding, layout_engine)
  File "/home/nicole/.local/lib/python3.9/site-packages/PIL/ImageFont.py", line 161, in __init__
    if core.HAVE_RAQM:
  File "/home/nicole/.local/lib/python3.9/site-packages/PIL/ImageFont.py", line 41, in __getattr__
    raise ImportError("The _imagingft C module is not installed")
ImportError: The _imagingft C module is not installed
```

i'm on Fedora 33 with xfce, on my notebook with just the same setup, i've no issue, but i can't find the correct setting i've forgotten to install.

**Comment by Gambloide** [comment]
I had the same problem under Fedora 33 and solved it by uninstalling pillow, installing the required build dependencies and then installing pillow again.

``` bash
# without sudo, the pillow uninstall silently failed
sudo pip3 uninstall pillow
sudo dnf install libjpeg-turbo-devel zlib-devel libtiff-devel openjpeg2-devel freetype-devel \
        lcms2-devel libwebp-devel tcl-devel tk-devel harfbuzz-devel fribidi-devel libraqm-devel \
        libimagequant-devel libxcb-devel
pip3 install pillow --no-cache-dir
```
My updated Fedora install script looks as follows.
**Notes:**
- I also removed the 'plugdev' references, as the 'plugdev' group is used in Debian and derivates only. Techpoet already raised a PR to fix this: #76 
- The explicit 'uaccess' tag might not be required.
- When installing pillow like this, the following error is displayed: `streamdeck-ui 1.0.2 requires pillow<7.0,>=6.1, but you'll have pillow 8.1.0 which is incompatible.` However, this does not seem to cause any issues and pillow seems to work as expected and no errors or warnings are shown in the terminal when running 'streamdeck' and changing images and texts of buttons.
- I can change button images and text, but the buttons do not do anything when pressed yet (neither commands, press-keys nor write-text work)

``` bash
#!/bin/bash -xe
echo "Installing libraries"
# installation dependencies
sudo dnf install python3-devel libusb-devel libusbx-devel libudev-devel systemd-devel
# execution dependencies
sudo pip3 uninstall pillow
sudo dnf install libjpeg-turbo-devel zlib-devel libtiff-devel openjpeg2-devel freetype-devel \
        lcms2-devel libwebp-devel tcl-devel tk-devel harfbuzz-devel fribidi-devel libraqm-devel \
        libimagequant-devel libxcb-devel
pip3 install pillow --no-cache-dir
echo "Adding udev rules and reloading"

sudo tee /etc/udev/rules.d/99-streamdeck.rules << EOF
SUBSYSTEM=="usb", ATTRS{idVendor}=="0fd9", ATTRS{idProduct}=="0060", MODE:="666", TAG+="uaccess"
SUBSYSTEM=="usb", ATTRS{idVendor}=="0fd9", ATTRS{idProduct}=="0063", MODE:="666", TAG+="uaccess"
SUBSYSTEM=="usb", ATTRS{idVendor}=="0fd9", ATTRS{idProduct}=="006c", MODE:="666", TAG+="uaccess"
SUBSYSTEM=="usb", ATTRS{idVendor}=="0fd9", ATTRS{idProduct}=="006d", MODE:="666", TAG+="uaccess"
EOF

sudo udevadm control --reload-rules

echo "Unplug and replug in device for the new udev rules to take effect"
echo "Installing streamdeck_ui"
pip3 install --user streamdeck_ui
echo "If the installation was successful, run 'streamdeck' to start."
```

**Comment by dodgyrabbit** [comment]
Closing as duplicate of #55. See the summary provided.

> neither commands, press-keys nor write-text work
Press keys and text won't work outside of streamdeck-ui app itself on Fedora, due to Wayland and lack of support in pynput.
The command feature should work, but likely due to lack of error handling in 1.0.2, the app likely stops working altogether.

See known issues and help in the readme. Hopefully 1.0.3 should avoid the most of the issues mentioned here.
